deploy/bicep/modules/storage.credentials.to.keyvault.bicep

```diff
@@ -24,3 +25,14 @@ module storageAccountKeySecret './akv.secrets.bicep' = {
     secretValue: storageAccount.listKeys().keys[0].value
   }
 }
+
+module storageAccountConnStrSecret './akv.secrets.bicep' = {
+  name: '${toLower(secretNamePrefix)}-storage-account-connstr-${utcValue}'
+  scope: resourceGroup(keyVaultResourceGroup)
+  params: {
+    environmentName: environmentName
+    keyVaultName: keyVaultName
+    secretName: storageAccountConnStrSecretNameVar
+    secretValue: 'DefaultEndpointsProtocol=https;AccountName=${storageAccount.name};EndpointSuffix=${environment().suffixes.storage};AccountKey=${listKeys(storageAccount.id, storageAccount.apiVersion).keys[0].value}'
```

Comment:
Is it possible to also retrieve this from the storageAccount resource so that we're not constructing the string? It would just make it a little less fragile in case the format of this string changes...

Reply:
Yeah, that's too bad. I looked around as well and couldn't find anything. Given that, what you have is the best approach.